flatdata-rs/lib/src/generator.rs

```diff
@@ -117,7 +117,7 @@ pub fn generate(
         );
 
         std::fs::create_dir_all(result.parent().unwrap())?;
-        let _ = std::process::Command::new(out_dir.join("bin/flatdata-generator"))
+        let success = std::process::Command::new(out_dir.join("bin/flatdata-generator"))
```

Comment:
Since we imported `std::process` already, you might want to remove this prefix.

Reply:
Done